Modules/Core/Mesh/include/itkTriangleMeshCurvatureCalculator.h

```diff
@@ -129,15 +131,12 @@ class ITK_TEMPLATE_EXPORT TriangleMeshCurvatureCalculator : public Object
 
   /** Discrete Gauss curvature (K) computation */
   void
-  GetGaussCurvature(MeshConstPointer input);
-
-  void
-  ComputeGaussCurvature(MeshConstPointer input);
+  ComputeGaussCurvature(const InputMeshType * input);
 
 private:
   CurvaturesEnum               m_CurvatureType = CurvaturesEnum::GaussCurvature;
   DoubleVectorContainerPointer m_GaussCurvatureData;
-  MeshConstPointer             m_TriangleMesh;
```

Comment:
This still needs to be a smart pointer (we want to hold a reference to the object).